docs/topics/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -0,0 +1,64 @@
+[#proc_deploying-an-application-to-openshift]
+= Deploying an Application to OpenShift
+
```

Comment:
Same here, an introduction would be nice (what application, what sort of deployment this is, etc.).